install_config/storage_examples/shared_storage.adoc

```diff
@@ -13,4 +13,473 @@ toc::[]
 == Overview
 The following use case describes how a user wanting to leverage shared storage for use by two seperate containers would configure the solution. This example highlights the use of NFS, but can easily be adapted to other shared storage types, such as GlusterFS. In addition, this example will show configuration of pod security as it relates to shared storage.
 
+For an explanation of persistent volumes (PVs), persistent volume claims (PVCs), and
+using NFS as persistent storage, please see the section on
+link:../persistent_storage/persistent_storage_nfsfs[Persistent Storage Using NFS].
+Below, is an end-to-end example of using an existing nfs cluster as an OpenShift persistent store.
+It is assumed that a working NFS server and exports  has already been setup. 
+
+= Sharing storage across two containers using NFS
+{product-author}
```

Comment:
some kind of error here since this is what it looks like in my view:
{product-author} {product-version} :data-uri: :icons: :experimental: :toc: macro :toc-title: :prewrap!: